acvm/Cargo.toml

```diff
@@ -30,8 +30,8 @@ indexmap = "1.7.0"
 thiserror = "1.0.21"
 
 [features]
-bn254 = ["acir/bn254"]
-bls12_381 = ["acir/bls12_381"]
+bn254 = ["acir/bn254", "stdlib/bn254"]
```

Comment:
Passing comment: similar to how we re-export acir from acvm and acir_field, we may want to eventually re-export stdlib too